I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 409, in script
    if feel != 0:
  File "game/script.rpy", line 409, in <module>
    if feel != 0:
NameError: name 'feel' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 409, in script
    if feel != 0:
  File "renpy/ast.py", line 1898, in execute
    if renpy.python.py_eval(condition):
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 409, in <module>
    if feel != 0:
NameError: name 'feel' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
A mess 1.0
Wed Mar 30 20:20:05 2022
